moveit_ros/hybrid_planning/test/CMakeLists.txt

```diff
@@ -1,10 +1,10 @@
 add_executable(cancel_action cancel_action.cpp)
-ament_target_dependencies(cancel_action ${THIS_PACKAGE_INCLUDE_DEPENDS})
-target_link_libraries(cancel_action ${LIBRARIES})
```

Comment:
Was this previously dereferencing a variable that didn't exist?

Reply:
It should not  be necessary to link against the target list in this variable